cmd/prometheus/config.go

```diff
@@ -190,45 +187,6 @@ func init() {
 		"Local storage engine. Supported values are: 'persisted' (full local storage with on-disk persistence) and 'none' (no local storage).",
 	)
 
-	// Remote storage.
-	cfg.fs.StringVar(
```

Comment:
(Also listed in our 1.0 blog post: https://prometheus.io/blog/2016/07/18/prometheus-1-0-released/)